src/App.js

```diff
@@ -2,22 +2,18 @@ import React, { Component } from 'react';
 import { Route, Switch, Link, NavLink } from "react-router-dom";
 import MovieDetails from './components/MovieDetails';
 import Home from './components/Home';
-// import Header from './components/Header';
+import Login from './components/Login';
 import About from './components/FooterLinks/About';
 import FAQ from './components/FooterLinks/FAQ';
 import ContactUs from './components/FooterLinks/ContactUs';
-import homeButton from './home.png';
+import homeButton from './Images/home.png';
 import { fetchMovieData } from './apiCalls'
 import './App.css';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      logIn: this.logIn.bind(this),
```

Comment:
So you got it to pass the methods down without this binding syntax? I was getting some weird errors when I first tried that way -- will have to talk to you about how you got it working.